src/parser/parser.py

```diff
@@ -0,0 +1,64 @@
+"""Parses a PDF document.
+"""
+import logging
+
+
+import slate
+
+
+logging.basicConfig(level=logging.INFO)
+
+
+class Article:
```

Comment:
Will `Article` have any methods defined on it, or is it just a container for fields? If it does not have any methods, consider using a [namedtuple][] instead.

[namedtuple]: https://docs.python.org/3.3/library/collections.html?highlight=namedtuple#collections.namedtuple